From 74769a98f2a3e7f4837d508332b0622fb7491105 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Mon, 14 Nov 2022 09:43:43 -0800
Subject: [PATCH 185/196] Update seeds lists (#295)

This was manually updated in several places. The seeders are strangly
not seeding a variety of nodes. There are only a few available. In
addition to the nodes which the seeder is dumping, I have manually
updated the list with peers from my own node, and the node which has
been running the explorer.
---
 contrib/seeds/makeseeds.py   |  2 +-
 contrib/seeds/nodes_main.txt |  8 ++++++--
 contrib/seeds/seeds_main.txt | 12 +++++++-----
 contrib/seeds/seeds_test.txt |  2 +-
 src/chainparamsseeds.h       |  8 ++++++--
 5 files changed, 21 insertions(+), 11 deletions(-)

diff --git a/contrib/seeds/makeseeds.py b/contrib/seeds/makeseeds.py
index bf6c80f9f..1d76ab17c 100755
--- a/contrib/seeds/makeseeds.py
+++ b/contrib/seeds/makeseeds.py
@@ -36,7 +36,7 @@ PATTERN_ONION = re.compile(
 
 # Used to only select nodes with a user agent string compatible with the
 # eCash network.
-PATTERN_AGENT = re.compile(r"^(/lotusd:1.0.(\d+)\(.+\)/)")
+PATTERN_AGENT = re.compile(r"^(/lotusd:3.(\d+).(\d+)\(.+\)/)")
 
 def parseline(line):
     sline = line.split()
diff --git a/contrib/seeds/nodes_main.txt b/contrib/seeds/nodes_main.txt
index 9a16f58ce..a50d801b0 100644
--- a/contrib/seeds/nodes_main.txt
+++ b/contrib/seeds/nodes_main.txt
@@ -1,3 +1,7 @@
 35.184.152.63:10605
-85.209.48.163:10605
-188.68.46.226:10605
+45.119.84.253:10605
+65.21.88.223:10605
+116.110.6.191:10605
+123.20.255.166:10605
+152.165.70.246:10605
+173.208.206.181:10605
diff --git a/contrib/seeds/seeds_main.txt b/contrib/seeds/seeds_main.txt
index 459f96710..ec9ec8d46 100644
--- a/contrib/seeds/seeds_main.txt
+++ b/contrib/seeds/seeds_main.txt
@@ -1,5 +1,7 @@
-35.184.152.63:10605                                 1   1634664743  100.00% 100.00% 100.00% 100.00%  98.18%   96402  00000405  70016 "/lotusd:1.0.0(EB32.0)/"
-188.68.46.226:10605                                 1   1634664743  100.00% 100.00% 100.00% 100.00%  98.18%   96402  00000405  70016 "/lotusd:1.0.1(EB32.0)/"
-85.209.48.163:10605                                 1   1634664743  100.00% 100.00% 100.00% 100.00%  98.18%   96402  00000405  70016 "/lotusd:1.0.1(EB32.0)/"
-194.36.145.176:10605                                1   1634664743  100.00% 100.00%  99.98%  99.91%  98.14%   96402  00000405  70016 "/lotusd:1.0.2(EB32.0)/"
-98.234.28.244:10605                                 0   1624385929   67.59%  24.55%   8.96%   1.33%   0.31%    3654  00000405  70016 "/lotusd:1.0.1(EB32.0)/"
\ No newline at end of file
+35.184.152.63:10605                                 1   1668437242  100.00% 100.00% 100.00% 100.00%  99.99%  379564  00000405  70016 "/lotusd:3.2.3(EB32.0)/"
+116.110.6.191:10605                                 1   1668437242  100.00% 100.00% 100.00% 100.00%  99.99%  379564  00000405  70016 "/lotusd:3.2.3(EB32.0)/"
+152.165.70.246:10605                                 1   1668437242  100.00% 100.00% 100.00% 100.00%  99.99%  379564  00000405  70016 "/lotusd:3.2.3(EB32.0)/"
+45.119.84.253:10605                                 1   1668437242  100.00% 100.00% 100.00% 100.00%  99.99%  379564  00000405  70016 "/lotusd:3.2.3(EB32.0)/"
+123.20.255.166:10605                                 1   1668437242  100.00% 100.00% 100.00% 100.00%  99.99%  379564  00000405  70016 "/lotusd:3.2.3(EB32.0)/"
+65.21.88.223:10605                                 1   1668437242  100.00% 100.00% 100.00% 100.00%  99.99%  379564  00000405  70016 "/lotusd:3.2.3(EB32.0)/"
+173.208.206.181:10605                                 1   1668437242  100.00% 100.00% 100.00% 100.00%  99.99%  379564  00000405  70016 "/lotusd:3.2.3(EB32.0)/"
\ No newline at end of file
diff --git a/contrib/seeds/seeds_test.txt b/contrib/seeds/seeds_test.txt
index 5f8e244e9..da4aed6a9 100644
--- a/contrib/seeds/seeds_test.txt
+++ b/contrib/seeds/seeds_test.txt
@@ -1 +1 @@
-35.224.106.240:11605                                1   1634664405  100.00% 100.00% 100.00% 100.00%  98.18%   11342  00000405  70016 "/lotusd:1.0.0(EB32.0)/"
\ No newline at end of file
+35.224.106.240:11605                                1   1668439482  100.00% 100.00% 100.00% 100.00%  99.99%  325116  00000405  70016 "/lotusd:3.2.3(EB32.0)/"
diff --git a/src/chainparamsseeds.h b/src/chainparamsseeds.h
index eac67c26d..d28a4d25a 100644
--- a/src/chainparamsseeds.h
+++ b/src/chainparamsseeds.h
@@ -9,8 +9,12 @@
  */
 static SeedSpec6 pnSeed6_main[] = {
     {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x23,0xb8,0x98,0x3f}, 10605},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x55,0xd1,0x30,0xa3}, 10605},
-    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xbc,0x44,0x2e,0xe2}, 10605}
+    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x2d,0x77,0x54,0xfd}, 10605},
+    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x41,0x15,0x58,0xdf}, 10605},
+    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x74,0x6e,0x06,0xbf}, 10605},
+    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x7b,0x14,0xff,0xa6}, 10605},
+    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x98,0xa5,0x46,0xf6}, 10605},
+    {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xad,0xd0,0xce,0xb5}, 10605}
 };
 
 static SeedSpec6 pnSeed6_test[] = {
-- 
2.39.5 (Apple Git-154)

